templates/_scheduler.erb

```diff
@@ -38,6 +38,9 @@
 <% if @authorization_kubeconfig then -%>
  --authorization-kubeconfig=<%= scope['kubernetes::master::scheduler::authorization_kubeconfig'] -%>
 <% end -%>
+<% if @authentication_kubeconfig then -%>
+ --authentication-kubeconfig=<%= scope['kubernetes::master::scheduler::authentication_kubeconfig'] -%>
```

Comment:
If you already have this variable in the scope (`@authentication_kubeconfig`) you don't need to use `scope['kubernetes::master::scheduler::authentication_kubeconfig']` here, just use `@authentication_kubeconfig`.